videostream.cpp

```diff
@@ -279,14 +387,15 @@ void cVideoStream::SetVideoSize(int width, int height)
 void cVideoStream::GetVideoSize(int *width, int *height, double *aspect_ratio)
 {
 	std::lock_guard<std::mutex> lock(m_mutex);
-	*width = 0;
-	*height = 0;
-	*aspect_ratio = 1.0f;
-
-	if (m_videoWidth && m_videoHeight) {
-		*width = m_videoWidth;
-		*height = m_videoHeight;
-		*aspect_ratio = (double)(m_videoWidth) / (double)(m_videoHeight);
+
+	if (m_pDecoder && m_pDecoder->GetContext()) {
+		*width = m_pDecoder->GetContext()->coded_width;
+		*height = m_pDecoder->GetContext()->coded_height;
+		*aspect_ratio = *width / (double)*height;
+	} else {
+		*width = 0;
+		*height = 0;
+		*aspect_ratio = 0.0;
```

Comment:
```suggestion
		*aspect_ratio = 1.0;
```
As per VDR documentation aspect_ratio is 1.0 per default.